docs/user-guide/area_calc.ipynb

```diff
@@ -6,9 +6,6 @@
    "source": [
```

Comment:
<p>Can you use latex to represent 4*pi? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>